faqs/atm7_http.md

```diff
@@ -0,0 +1,27 @@
+---
+title: How to enable HTTP on >1.12
+search: atm7 http
+---
+
+**Singleplayer**
+1. In CurseForge, go to "My Modpacks" and right-click the ATM-7 modpack.
+2. Select "Open Folder" in the dropdown menu.
+3. Open the folder titled `saves`.
+4. Find your world folder, and open it.
+5. Find a folder titled `serverconfig`, and open it.
+6. Search for a file named `computercraft-server.toml`, and open it in a text editor.
+7. Search the file for a section starting with `[http]`.
+8. Find `enabled` within this section, and set it to `true`.
+9. Save the file and close the editor, then save and quit to title (or `stop` the server), then relaunch the world.
```

Comment:
They really should turn game off before editing config. In some cases if you edit config with running game it won't stick. Also why are you talking about stopping server if this is single player faq?

Reply:
I think it was meant to be redirected to from the multiplayer server owner thing. But its not really the same.